GliaWidgets/Component/CallButtonBar/Button/CallButtonStyle.swift

```diff
@@ -82,19 +82,8 @@ public struct CallButtonStyle {
 
         }
 
-        button?.active?.title?.font.map { titleFont in
-            if let fontSize = titleFont.size, let fontStyle = titleFont.style {
-                switch fontStyle {
-                case .bold:
-                    active.titleFont = UIFont.boldSystemFont(ofSize: fontSize)
-                case .italic:
-                    active.titleFont = UIFont.italicSystemFont(ofSize: fontSize)
-                case .regular:
-                    active.titleFont = UIFont.systemFont(ofSize: fontSize)
-                case .thin:
-                    active.titleFont = UIFont.systemFont(ofSize: fontSize, weight: .thin)
-                }
-            }
+        if let font = UIFont.convertToFont(font: button?.active?.title?.font) {
```

Comment:
I'm not sure that it is significantly better way, however, we can handle optionality in this way.
```swift
UIFont.convertToFont(font: button?.active?.title?.font)
    .map { active.titleFont = $0 }
```

Reply:
You're totally right. But I think it was bad practice from me to try to refactor approved code from an unrelated branch. As mentioned in the [https://github.com/salemove/ios-sdk-widgets/pull/376#issuecomment-1254886413](comment), I will create a separate PR for those fixes, which would have a wider range than this single fix.